action.yml

```diff
@@ -103,7 +103,49 @@ runs:
           rm tailscale.tgz
           TSPATH=/tmp/tailscale_${VERSION}_${TS_ARCH}
           sudo mv "${TSPATH}/tailscale" "${TSPATH}/tailscaled" /usr/bin
-      - name: Start Tailscale Daemon
+      - name: Download Tailscale - Windows
+        if: ${{ runner.os == 'Windows' }}
+        shell: bash
+        env:
+          VERSION: ${{ inputs.version }}
+          SHA256SUM: ${{ inputs.sha256sum }}
+        run: |
+          if [ "$VERSION" = "latest" ]; then
+            VERSION=$(curl -s "https://pkgs.tailscale.com/stable/?mode=json" | jq -r .Version)
```

Comment:
I noticed you used `-H user-agent:tailscale-github-action` in a `curl` further down. Want to do it here too?